src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/RestoreBarLogger.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.ProjectManagement;
+
+namespace NuGet.PackageManagement.UI.Utility
+{
+    internal class RestoreBarLogger : LoggerBase
+    {
+        private readonly INuGetProjectContext _nuGetProjectContext;
+
+        public RestoreBarLogger(INuGetProjectContext nuGetProjectContext)
+            : base(verbosityLevel: LogLevel.Warning)
```

Comment:
☝️ Key point: Log level is set to a higher level here. Leverages the existing LoggerBase as other loggers do.

Reply:
This would be a fine thing to put in an inline comment.

```C#
// The log level is `Warning` so that ...
```

Reply:
Done!

Reply:
Great comment!